src/vm/core_test.zig

```diff
@@ -3609,3 +3610,94 @@ test "CairoVM: verifyAutoDeductionsForAddr throws InconsistentAutoDeduction" {
 
     try expectError(CairoVMError.InconsistentAutoDeduction, vm.verifyAutoDeductionsForAddr(allocator, Relocatable.init(2, 2), &builtin));
 }
+
+test "CairoVM: verifyAutoDeductions for bitwise builtin runner" {
+    const allocator = std.testing.allocator;
+
+    var bitwise_builtin = BitwiseBuiltinRunner.initDefault();
+    bitwise_builtin.base = 2;
+    const builtin = BuiltinRunner{ .Bitwise = bitwise_builtin };
+
+    var vm = try CairoVM.init(allocator, .{});
+    defer vm.deinit();
+
+    try vm.builtin_runners.append(builtin);
+
+    try vm.segments.memory.setUpMemory(
+        allocator,
+        .{
+            .{ .{ 2, 0 }, .{12} },
+            .{ .{ 2, 1 }, .{10} },
+            .{ .{ 2, 2 }, .{8} },
+        },
+    );
+    defer vm.segments.memory.deinitData(allocator);
+
+    const result = try vm.verifyAutoDeductions(allocator);
+    try expectEqual(void, @TypeOf(result));
+}
+
+test "CairoVM: verifyAutoDeductions for bitwise builtin runner throws InconsistentAutoDeduction" {
+    const allocator = std.testing.allocator;
+
+    var bitwise_builtin = BitwiseBuiltinRunner.initDefault();
+    bitwise_builtin.base = 2;
+    const builtin = BuiltinRunner{ .Bitwise = bitwise_builtin };
+
+    var vm = try CairoVM.init(allocator, .{});
+    defer vm.deinit();
+
+    try vm.builtin_runners.append(builtin);
+
+    try vm.segments.memory.setUpMemory(
+        allocator,
+        .{
+            .{ .{ 2, 0 }, .{12} },
+            .{ .{ 2, 1 }, .{10} },
+            .{ .{ 2, 2 }, .{7} },
+        },
+    );
+
+    defer vm.segments.memory.deinitData(allocator);
+
+    try expectError(CairoVMError.InconsistentAutoDeduction, vm.verifyAutoDeductions(allocator));
+}
+
+test "CairoVM: verifyAutoDeductions for keccak builtin runner" {
+    const allocator = std.testing.allocator;
+
+    var keccak_instance_def = try KeccakInstanceDef.default(allocator);
```

Comment:
This was converted to `initDefault` recently to respect conventions:

https://github.com/keep-starknet-strange/ziggy-starkdust/blob/103e0543534decd5f17267202e178e7288283c38/src/vm/types/keccak_instance_def.zig#L33